PS8lite.Rmd

```diff
@@ -177,33 +178,53 @@ MiscVal + MoSold + YrSold" %>%
 ```
 
 
-
 ***
-
-
+# View train data 
+```{r, eval=FALSE}
+View(training)
+```
```

Comment:
Hi, everything looks good. Here is an example comment that I would leave as part of a “code review” if this were real life. Then you would make the requested changes to this PR and then I would merge your changes into my master copy of the repo. I won't be doing that last step in this case.